sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/implementation/oauth2/AbstractOAuth2AuthorizationCodeGrantRequestEntityConverter.java

```diff
@@ -57,6 +48,6 @@ public HttpHeaders getHttpHeaders() {
      * @return MultiValueMap
      */
     public MultiValueMap<String, String> getHttpBody(OAuth2AuthorizationCodeGrantRequest request) {
-        return null;
+        return new LinkedMultiValueMap<>();
```

Comment:
My personal preference would be to return an immutable collection here, if possible.  Hence my suggestion of `new MultiValueMapAdapter<>(Collections.emptyMap());`

Maybe even have a static field to return, so there is no repeated allocation.